polaris/ocean/tests/baroclinic_channel/baroclinic_channel_test_case.py

```diff
@@ -35,7 +37,7 @@ def __init__(self, test_group, resolution, name):
         if resolution >= 1.:
             res_str = f'{resolution:g}km'
         else:
-            res_str = f'{resolution:g}m'
+            res_str = f'{resolution * 1000.:g}m'
```

Comment:
@alicebarthel, if this looks right (and you have no further concerns), go ahead and approve the PR.